hrpsys_gazebo_general/package.xml

```diff
@@ -17,15 +17,16 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>hrpsys_ros_bridge</build_depend>
-  <build_depend>hrpsys_gazebo_msgs</build_depend>
-  <build_depend>gazebo_ros</build_depend>
   <build_depend>collada_urdf_jsk_patch</build_depend>
+  <build_depend>gazebo_ros</build_depend>
+  <build_depend>hrpsys_gazebo_msgs</build_depend>
+  <build_depend>hrpsys_ros_bridge</build_depend>
```

Comment:
Order治すのと、build追加するのを別コミットにしましょう

--
◉ Kei Okada

2016/04/18 23:53、iory <notifications@github.com> のメッセージ:

> In hrpsys_gazebo_general/package.xml:
> 
> >    <build_depend>collada_urdf_jsk_patch</build_depend>
> > +  <build_depend>gazebo_ros</build_depend>
> > +  <build_depend>hrpsys_gazebo_msgs</build_depend>
> > +  <build_depend>hrpsys_ros_bridge</build_depend>
> alphabet orderです．
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>